content/applications/inventory_and_mrp/manufacturing/plm/version_control.rst

```diff
@@ -0,0 +1,138 @@
+select the desired |BOM|. On the |BOM|, navigate to the *chatter* and click the :guilabel:`📎
+(paperclip)` icon. The files associated with the |BOM| are displayed in the :guilabel:`Files`
+section. To add more design files, select the :guilabel:`Attach files` button.
+
+.. image:: version_control/attach-files.png
+    :align: center
+    :alt: Show paperclip icon in the chatter to attach files to a BoM.
+
+Manage design files in an ECO
+-----------------------------
+
+After creating a :ref:`new Engineering Change Order <plm/eco/create-eco>` (ECO), all files stored
+in the :guilabel:`Files` section of the |BOM| are automatically copied over to the
+:guilabel:`Documents` smart button. Click on the button to open the :guilabel:`Attachments` page.
+
+Hover over each attachment to reveal the :guilabel:`︙ (three vertical dots)` icon. From there,
+choose whether to :guilabel:`Edit`, :guilabel:`Remove`, or :guilabel:`Download` the file. Any
+changes made to these files are contained within the |ECO|, and will only apply to the production
+|BOM| once the :ref:`changes are applied <plm/eco/apply-changes>`.
+
+.. example::
+   Replace the keyboard PDF in an active |ECO| by first selecting the :guilabel:`Documents` smart
+   button.
+
+   .. image:: version_control/documents-smart-button.png
+      :align: center
+      :alt: Show *Documents* smart button from an active ECO.
+
+   On the :guilabel:`Attachments` page, hover over the design file and click the :guilabel:`︙ (three
+   vertical dots)` icon, then the :guilabel:`Edit` option. Rename the file something specific to
+   express why the file is about to be archived in this active |ECO|. In this case, the |BOM| is for
+   a smaller keyboard, so the manual for a larger keyboard is no longer relevant. Rename `100%
+   keyboard manual`, and then save and click back to the :guilabel:`Attachments` page using the
+   breadcrumbs. Click on the :guilabel:`︙ (three vertical dots)` icon again, and then select the
+   :guilabel:`Remove` option to archive the file.
+
+   Next, click the :guilabel:`Upload` button to upload the new design file, named `60% keyboard
+   manual`.
+
+   .. image:: version_control/attachments.png
+      :align: center
+      :alt: View of *Attachments* page from the *Documents* smart button. Displays one archived and
+            one newly added attachment.
+
+.. note::
+   Archived files are not permanently deleted—they can still be accessed in the previous |ECO| or as
+   an archived file in the latest ECO where the archival occurred.
+
+Apply rebase
+============
+
+Odoo simplifies the process of merging different versions of a |BOM| when multiple |ECOs| are open
+for the same product. If the |BOM| used in production is updated while an |ECO| is still open and in
+testing, review the differences between the production |BOMs| in the newly appeared
+:guilabel:`Previous Eco Bom Changes` tab. Then, click the :guilabel:`Apply Rebase` button to update
```

Comment:
Is is `Eco` or should it be `ECO`? Disregard if I already commented this before :-)

Reply:
I want it to be ECO, but on the UI, it says `Eco` and `Bom` :cry: